run-skynet-requests.py

```diff
@@ -0,0 +1,89 @@
+# Description: This script is used to run skynet requests from a dump file
+# Usage: poetry run python -m run-skynet-requests -f <skynet-jobs-dump-file> -m <max-requests> -s <sleep-time> -u <skynet-url>
+# Prerequisites: skynet jobs dump file and a running skynet instance
+
+import asyncio
+import json
+import aiohttp
+
+from argparse import ArgumentParser
+from tqdm import tqdm
+
+
+parser = ArgumentParser()
+parser.add_argument('-f', '--file', dest='filename', help='skynet jobs dump file ', metavar='FILE')
+parser.add_argument('-m', '--max', dest='max_requests', help='max requests to make', default=10)
+parser.add_argument('-s', '--sleep', dest='sleep', help='sleep time in seconds. Should be an estimate of how long max_requests needs to be processed', default=600)
+parser.add_argument('-u', '--url', dest='url', help='skynet url', default='http://localhost:8000')
+parser.add_argument('-t', '--type', dest='type', help='job type', default='any')
+
+args = parser.parse_args()
+
+jobs = open(args.filename, 'r').readlines()
+
+if args.type != 'any':
+    jobs = [job for job in jobs if json.loads(job)['type'] == args.type]
+
+max_requests = int(args.max_requests)
+base_url = args.url
+sleep_time = int(args.sleep)
+
+
+async def main():
+    session = aiohttp.ClientSession()
+
+    async def post(job_type, data):
```

Comment:
Does it make sense to add jwt support to test it against stage for example?

Reply:
it does, i actually wanted to do that but kinda forgot, ty for reminding me